cypress/e2e/tests/pages/cluster-manager.spec.ts

```diff
@@ -14,10 +16,14 @@ const runPrefix = `e2e-test-${ runTimestamp }`;
 
 // File specific consts
 const { baseUrl } = Cypress.config();
-const clusterRequestBase = `${ baseUrl }/v1/provisioning.cattle.io.clusters/fleet-default`;
+const namespace = 'fleet-default';
+const type = 'provisioning.cattle.io.cluster';
+const clusterRequestBase = `${ baseUrl }/v1/${ type }/${ namespace }`;
 const clusterNamePartial = `${ runPrefix }-create`;
 const rke2CustomName = `${ clusterNamePartial }-rke2-custom`;
 const importGenericName = `${ clusterNamePartial }-import-generic`;
+const clusterNameImport = `${ clusterNamePartial }-${ runTimestamp }-import`;
```

Comment:
With the changes below both `clusterNameImport` and `clusterName` can be removed